qutebrowser/utils/utils.py

```diff
@@ -864,6 +864,63 @@ def unused(_arg):
     pass
 
 
+def parse_numsettxt_into_list(txtset, nummax):
+    """Parse the given numbers set in text format, and return it as a list of
+    numbers.
+
+    Args:
+        txtset: Numbers set specification. E.g. '1,5,8-last,32,2', where 'last'
+                is a special keyword that denotes that maximum allowable
+                number.
+        nummax: The number that will be used to replace 'last'
+    """
+    txtset = txtset.lower().replace('last', '%d' % (nummax))
+    l = set()
+    parts = txtset.split(',')
+    for part in parts:
+        if part.find('-') != -1:
+            [start, end] = part.split('-')
+            start = int(start)
+            end = int(end)
+            for i in range(int(start), int(end)+1):
+                l.add(i)
+        else:
+            l.add(int(part))
+    return sorted(l)
+
+
+def parse_list_into_numsettxt(l):
```

Comment:
I'm really not sure if the complexity of this is worth it. I think it'd be better to just return a comma-separated list, even if it's `1,2,3`. Just seems like a minor benefit for something which isn't immediately obvious when looking at the code, so I'm inclined to drop it.